run.py

```diff
@@ -36,21 +38,25 @@ def _setup_out_dir(self):
         return xunit_dir
 
     def _get_xunit_file(self, xunit_dir):
-        return os.path.join(xunit_dir, 'nosetests.{}.xml'.format(self._tag))
+        return os.path.join(xunit_dir, 'nosetests.v{}.{}.xml'.format(self._protocol, self._tag))
 
     def _ignoreFile(self):
         here = os.path.dirname(__file__)
         return os.path.join(here, 'versions', self._tag, 'ignore.yaml')
 
     def _ignoreSet(self):
+        ignore_tests = []
         with open(self._ignoreFile()) as f:
             content = yaml.load(f)
-            return set(content['tests'])
+            ignore_tests.extend(content['tests'])
+            if self._protocol == '4' and 'v4_tests' in content and content['v4_tests']:
+                ignore_tests.extend(content['v4_tests'])
+        return set(ignore_tests)
```

Comment:
why did we need this change - I don't think you have placed any tests under v4_tests as ignored .... or are they missing from this commit ?

I agree that we will probably need it because of the special addition they had for v4 - but if we haven't found such tests lets defer the addition till its needed.